### DISCUSSION TITLE
stoicism/cormac/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/stoicism#14] [General]
# stoicism/cormac/

https://shanenull.com/stoicism/cormac/

**Comment by shane0** [comment]
# sunset limited

```text
Your God must once have stood
at a dawn of infinite
possibilities,
and that is what
he's made of it.
```

- I don't think there's anything wrong with me I think I've just been driven to finally face the truth
- [clip](https://www.youtube.com/clip/UgkxV3VM6sU_o-TdmrV-QTvcj05i5MNISByx)

- I don't regard my state of mind as some pessimestic view of the world I regard it as the world itself
- [clip](https://www.youtube.com/clip/UgkxBWceB9zd38KA3fGU9JEoOX6nsdsNvHRv)

**Reply by shane0** [reply]
“All right. It’s that the world is basically a forced labor camp from which the workers—perfectly innocent—are led forth by lottery, a few each day, to be executed. I dont think that this is just the way I see it. I think it’s the way it is. Are there alternate views? Of course. Will any of them stand close scrutiny? No.”

Excerpt From
The Sunset Limited
Cormac McCarthy

**Reply by shane0** [reply]
```text
The darker picture is always the correct one. When you read the history of the world you are reading a saga of bloodshed and greed and folly the import of which is impossible to ignore. And yet we imagine that the future will somehow be different.
From The Sunset Limited by Cormac McCarthy
```

**Reply by shane0** [reply]
```text
When you read the history of the world you are reading a saga of bloodshed and greed and folly the import of which is impossible to ignore. And yet we imagine that the future will somehow be different.
From The Sunset Limited by Cormac McCarthy
```

**Reply by shane0** [reply]
## mashup

```text
I don't regard my state of mind as some pessimistic view of the world I regard it as the world itself
I don't think there's anything wrong with me I think I've just been driven to finally face the truth
The darker picture is always the correct one
When you read the history of the world you are reading a saga of bloodshed and greed and folly the import of which is impossible to ignore
And yet we imagine that the future will somehow be different
the world is basically a forced labor camp from which the workers — perfectly innocent — are led forth by lottery, a few each day, to be executed
I don't think that this is just the way I see it
I think it’s the way it is
Are there alternate views? Of course.
Will any of them stand close scrutiny? No


the dystopia is subsidized they buy us with our own money and  use it for engineering ongoing drug problems war pollution and healthcare issues then blame it on us and tax us or subsidize their fake ongoing solutions



```

**Reply by shane0** [reply]
https://youtu.be/3SUsdGqb2lU?si=y-MhA5NUAQPYrxpv

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/docs_2021/sadhana/sunset/#sunset-unlimited

**Reply by shane0** [reply]
My own reasons center around
a gradual loss
of make-believe,
a gradual enlightenment
as to the nature
of reality, of the world.

**Reply by shane0** [reply]
You give up the world
line by line.
You become an accomplice
to your own annihilation.
There's nothing
you can do about it.
Everything you do closes a
door somewhere ahead of you.
Finally there's
only one door left.

**Reply by shane0** [reply]
https://shanenull.com/buddhism/2021/docs_2021/sadhana/sunset/

**Reply by shane0** [reply]
## books

- https://www.abebooks.com/servlet/SearchResults?ch_sort=t&cm_sp=sort-_-SRP-_-Results&kn=cormac%20mccarthy%20books&sortby=1

**Comment by shane0** [comment]
pretty horses
https://vimeo.com/308804948

**Reply by shane0** [reply]
https://youtu.be/A3HyI2BQim4?si=rKHBSOWggOhlV9Ch

Don’t chew on something that’s trying to eat you

It’s good God hides the truth from children otherwise they wouldn’t have the heart to start

He forgot how stupid pain makes you should need the opposite otherwise what good is it

**Reply by shane0** [reply]
- losing coltrane

```text
you give up the world line by line you become an accomplice to your own annihilation
there is nothing you can do about it everything you do closes a door somewhere ahead of you
finally there is only one door left
```

https://youtube.com/clip/UgkxENIb7JDn7IaiWQNfuVA4lj0luN6YZJn7

**Comment by shane0** [comment]
https://github.com/shane0/stoicism/issues/13

**Reply by shane0** [reply]
```text
“Ultimately there is nothing to know and no one to know it.”
― Cormac McCarthy, The Passenger
```

**Reply by shane0** [reply]
- the road

```text
“I WAS AFRAID I WAS GOING TO DIE AND THEN I WAS AFRAID I WASNT"
```

**Reply by shane0** [reply]
```text
The problem is that what drives the tale will not survive the tale. As the room dims and the sound of voices fades you understand that the world and all in it will soon cease to be.
```

**Reply by shane0** [reply]
```text
above all and lastly the world does not know that you were here
```

**Reply by shane0** [reply]
# Harold Bloom

https://youtu.be/1cuccco2umo?si=q-0Ymzf8SwQAtC1H

Could not read blood meridian the first two attempts it takes a strong stomach then it’s rewarding

Has read it 20 to 30 times

**Comment by shane0** [comment]
Chaconne

https://youtu.be/XVC9bRDniOk?si=JivF0SMQ1-HNeQhz

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2024/quotes/

**Comment by shane0** [comment]
recorder https://youtu.be/m33P9pjBB0g

**Reply by shane0** [reply]
flute https://youtu.be/bTDO8wtH7Fs

**Comment by shane0** [comment]
https://shanenull.com/buddhism/2024/maranasati/

**Comment by shane0** [comment]
# outer dark

```text
water in the summer and fire in the winter is all the need i need - outer dark p. 235
```

**Comment by shane0** [comment]
# the counsellor

> The extinction of all reality is a concept that no resignation can encompass. And yet, in that despair, which is transcendent, you will find the ancient understanding that the Philosopher’s Stone will always be found despised and buried in the mud. This may seem like a small thing in the face of annihilation, until annihilation occurs. And then all the designs and grand plans will be revealed for what they are.

**Reply by shane0** [reply]
# dying

## jefe

> "Yes. At the understanding that life will not take you back. I have no wish to paint the world in colors more somber than those it wears, but as the world gives way to darkness it becomes more and more difficult to dismiss the understanding that the world is in fact oneself. It is a thing which you have created, no more, no less. And when you cease to be so will the world. There will be other worlds. Of course. But they are the worlds of other men and your understanding of them was never more than an illusion anyway. Your world—the only one that matters—will be gone. And it will never come again. The extinction of all reality is a concept no resignation can encompass. Until annihilation comes. And all grand ideas are seen for what they are. And now I must go. I have calls to make, and then, if there is time, I will take a little nap."
>> —The Counselor



* "I would urge you to see the truth of your situation, Counselor. That is my advice. It is not for me to say what you should have done."
* "Actions create consequences which produce new worlds."
* "The choosing was done a long time ago."
* "There will be other worlds. Of course. But they are the worlds of other men and your understanding of them was never more than an illusion anyway. Your world-- the only one that matters-- will be gone. And it will never come again."
* "Perhaps you would not agree. I don't know. But nothing is crueller than a coward, and the slaughter to come is probably beyond our imagining."

## others

* From "The Road" (2006):

 > "When he woke in the woods in the dark and the cold of the night he'd reach out to touch the child sleeping beside him. Nights dark beyond darkness and the days more gray each one than what had gone before. Like the ocean, a thing you could see if you went to it. You could say 'the ocean.' At night its waves—the darkness—came to you and you could hear them."

* From "No Country for Old Men" (2005):

**Reply by shane0** [reply]
## avoiding tech lock in

- own your audiobooks
- https://www.downpour.com/catalogsearch/result/?q=cormac#q=cormac%20mccarthy&idx=live_downpour_en_en_products&p=0&is_v=1
- https://audiobookstore.com/authors/cormac-mccarthy-audiobooks/
- https://www.audiobooks.com/search/book/cormac%20mccarthy

**Reply by shane0** [reply]
audiobooks.com is swedish or uk other is some llc

**Reply by shane0** [reply]
> A man would give entire nations to lift grief off his heart. And yet, you cannot buy anything with grief, because grief is worthless.

**Comment by shane0** [comment]
# similar to cormac

 > "He was a man who had been born in a bad time for men. Born to lose. And he had lost. And he had been born in a bad time for men and for nations."

```text
The world is made of flesh and blood and bone and the things that we do to each other. 
And the things that we do to ourselves.
You're a man who's made a great many mistakes. 
And you're going to make a few more.
```

**Comment by shane0** [comment]
# parallels

- orchard tree foreshadow easter eggs in first chapter are two of his book titles

> “For some time now the road had been deserted, white and scorching yet, though the sun was already reddening the western sky. ”


- in the road the wife is suicidal same as alicia western in stella marris

People were always getting ready for tomorrow. I didnt believe in that. Tomorrow wasnt getting ready for them. It didnt even know they were there.

**Reply by shane0** [reply]
orchard keeper main character goes to asylum? compare to alicia at stella marries

kekule problem is in every book

orchard keeper mentions outer dark

“his eyes shifting among them or when they looked closing or seeking again that being in the outer dark with whom only he held communion”

Excerpt From
The Orchard Keeper
Cormac McCarthy

**Comment by shane0** [comment]
# the rule

- common thread in all books

| **Book**                  | **Narrator or 'Rule-Giver' Presence**                                                                               | **Key Character Rule-Centered Quote**                                                                                                  |
|:--------------------------|:--------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------------------:|
| **Blood Meridian**        | Judge Holden acts as a philosophical narrator, lecturing on violence, law, and “the order of the world.”            | “Moral law is an invention of mankind for the disenfranchisement of the powerful in favor of the weak.”                                 |
| **No Country For Old Men**| Sheriff Bell’s narration offers moral commentary; Chigurh enforces his own personal code.                           | Bell: “You can’t stop what’s coming. It ain’t all waiting on you. That’s vanity.”<br>Chigurh: “If the rule you followed brought you to this, of what use was the rule?” |
| **The Road**              | The father as a moral guide, creating rules for the boy—‘carry the fire.’                                           | “You have to carry the fire.”<br>Father: “The fire inside you. It always was there.”                                                    |
| **Suttree**               | Third-person narration, with Suttree and others reflecting on society’s and personal survival rules.                 | “If he is not careful he will drown in all their voices. Must be his own. Must be a law unto himself.” (paraphrased)                   |
| **The Crossing**          | Old men, priests, and the wild act as sources of rules and philosophies for Billy Parham.                           | “He said that each man is the last man and that to permit oneself to fail is to fail eternally.”                                       |
| **The Passenger** / **Stella Maris** | Alicia and Bobby face existential and metaphysical boundaries; Dr. Cohen discusses scientific/moral rules.   | Alicia: “The only thing I know is that there are rules. And they cannot be violated even by those who made them.”                      |
| **All the Pretty Horses** | Rules of honor, tradition, and fate; elders articulate changing codes of the world.                                  | “Among men there was no such communion as among horses and the notion that men can be understood at all was probably an illusion.”     |
